Azure/azure-cli
# [App Service] `az webapp up`: Fix `--logs` arg fails with an exception

**Related command**

`az webapp up --logs`

**Description**

The command `az webapp up` fails if the `--logs` arg is passed. This is because logstreaming does not start working until the main site has been started and therefore, we get an exception while trying to stream logs. To fix this, we can ping the site to warmup prior to starting the logstreaming thread. 
Related issue: https://github.com/Azure/azure-cli/issues/23058

**History Notes**


[App Service] `az webapp up`: Fix `--logs` arg fails with an exception

**Testing Guide**


Test deploying webapps using `az webapp up --logs` 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
